.ruby-version

```diff
@@ -1 +1 @@
-2.6.5
+2.7.0
```

Comment:
Should just get rid of this file? Or get rid of `.tool-versions`? It feels weird to me having both.

Reply:
`.tool-versions` is just for asdf (my version manager). Since asdf can read `.ruby-version`, and so can everything else, we should standardize on `.ruby-version`. I'll remove `.tool-versions.